components/feature/top-sites/src/main/java/mozilla/components/feature/top/sites/PinnedSitesUseCases.kt

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.feature.top.sites
+
+/**
+ * Contains use cases related to the pinned sites feature.
+ */
+class PinnedSitesUseCase(storage: PinnedSiteStorage) {
```

Comment:
Shouldn't this be the top sites storage (the wrapper around the pinned and history)?

And as a result, the use cases would also be top sites use cases.